pkgs/servers/monitoring/prometheus/klipper-exporter.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nixosTests,
+}:
+
+buildGoModule rec {
+  pname = "prometheus-klipper-exporter";
+  version = "0.11.2";
+
+  src = fetchFromGitHub {
+    owner = "scross01";
+    repo = pname;
```

Comment:
Using pname for the repo name is discouraged, as it introduces an unnecessary coupling between the derivation name and the repo.

Could you just use the literal value here?